sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -625,12 +480,11 @@ object Interpreter {
 
   /** Result of fullReduction to sigma tree with costing. */
   case class FullReductionResult(
```

Comment:
This class looks like excessive meaningless wrapping now

Reply:
Will be removed in subsequent PRs